README.md

```diff
@@ -75,22 +77,22 @@ yarn prod
 1. Sign in with your credentials.
 1. Manage content in the content administration interface of your sample project.
 
-You can learn more about content editing with Kentico Kontent in the documentation.
+You can learn more about content editing with Kontent.ai in the documentation.
```

Comment:
I would just add a link to kontent.ai/learn here

Reply:
90d98d8